localdev/Dockerfile

```diff
@@ -0,0 +1,19 @@
+## This Dockerfile is used for building Horust in a conatiner while developing locally.
+## It follows https://blog.mgattozzi.dev/caching-rust-docker-builds/ AND SHOULD NOT BE USED IN PRODUCTION
+
+FROM rust:1.42
+WORKDIR /usr/src/myapp
+ARG CARGO_PARAMS
+# The dummy file is just used to get the deps into our project
+# Note that COPY only runs if the files it is provided has changed, otherwise the Docker cache is hit
+COPY Cargo.toml Cargo.lock ./
+# This uses the dummy Rust program to compile all deps
+RUN echo 'fn main() {}' > dummy.rs
```

Comment:
this should be src/dummy.rs I think

Reply:
It's running inside `WORKDIR`, so it doesn't matter I think

Reply:
Changed it to `src/dummy.rs`, and also added an empty file there to make `rustfmt` happy.

All logic is done in the `Dockerfile`, and once https://github.com/FedericoPonzi/Horust/issues/42 is fixed (and `ignore` in `rusfmt.toml` is actually used) we can get rid of the empty `dummy.rs`.

Reply:
Nope, no go, now the test scripts are angry at me:

```
  Compiling horust v0.1.1 (/home/runner/work/Horust/Horust)
error[E0601]: `main` function not found in crate `horust`
##[error] --> src/dummy.rs:2:75
  |
2 | // Remove once https://github.com/FedericoPonzi/Horust/issues/42 is fixed
  |                                                                           ^ consider adding a `main` function to `src/dummy.rs`

error: aborting due to previous error
```

I'm going to scrape the logic from the `Dockerfile`. If you have a suggestion on how to not get left with a `dummy.rs` file for now lmk.

Reply:
Resolving since we're not using a `dummy.rs` file anymore.

Reply:
Irrelevant, resolving